dracut/30ignition/ignition-setup-user.service

```diff
@@ -5,20 +5,11 @@ ConditionPathExists=/etc/initrd-release
 DefaultDependencies=false
 Before=ignition-complete.target
 
+After=boot.mount
```

Comment:
I think in general it's cleaner to use:
`RequiresMountsFor=/boot`
because that handles nested mounts as well as the case where there isn't a mount point there at all.  (Maybe someone in the future will want `/boot == /`)